src/EnergyPlus/SolarShading.cc

```diff
@@ -5970,7 +5970,24 @@ namespace SolarShading {
 					// The inside reveals receive solar (reflected part + absorbed part) from the window, this amount should be
 					// deducted from the BTOTZone, then adds the InsRevealDiffIntoZone
 					if ( SurfaceWindow( SurfNum ).WindowModelType == WindowBSDFModel ) { //Complex Fenestration
-						SurfSolIncPtr = SurfaceScheduledSolarInc( BackSurfaceNumber, ConstrNumBack );
+						// Jason Glazer added the following code and note on March 16, 2015 to address Github #4777
+						//
+						// Originally this just had the line:
+						//
+						//   SurfSolIncPtr = SurfaceScheduledSolarInc( BackSurfaceNumber, ConstrNumBack );
+						//
+						// prior to the line: if ( SurfSolIncPtr == 0 ) {
+						//
+						// But since BackSurfaceNumber, ConstrNumBack have not been set, added the code to scan through the
+						// back surfaces to look for the value. Since there can be more than one back surface for each, just
+						// loop through it and use the final value.
+						SurfSolIncPtr = 0;
+						int NBkSurf = ShadowComb( SurfNum ).NumBackSurf;
+						for ( int IBack = 1; IBack <= NBkSurf; ++IBack ) {
+							BackSurfaceNumber = ShadowComb( SurfNum ).BackSurf( IBack );
+							ConstrNumBack = Surface( BackSurfaceNumber ).Construction;
+							SurfSolIncPtr = SurfaceScheduledSolarInc( BackSurfaceNumber, ConstrNumBack );
```

Comment:
Just reading the IORef has me scratching my head on what to do here. Note "to the inside of this surface".
Field: Surface Name
The building surface associated with this object (ref BuildingSurface:Detailed). Solar absorptance values in the schedule file will be applied to the inside of this surface.

Does that mean add to the back surface of this surface (and therefore not to the zone)? Or is the documentation wrong? Is this is a way to trick the surface into having a window with solar gains being scheduled? If so, then solar wouldn't land on this surface at all since gains would pass through a window and land on a receiving surface. Do these scheduled gains apply to a receiving surface? What I don't understand is if the SurfSolIncPtr == 0, then there are no solar gains to be backed out of zone gains (i.e., shouldn't it be if ( SurfSolIncPtr > 0 )). 

Mike's suggestion makes sense, with what little I have understood what should happen.